src/fides/api/ctl/routes/system.py

```diff
@@ -350,3 +358,41 @@ async def create(
         await db.refresh(created_system)
 
     return created_system
+
+
+@system_router.get(
+    "/",
+    dependencies=[
+        Security(
+            verify_oauth_client_prod,
+            scopes=[SYSTEM_READ],
+        )
+    ],
+    response_model=List[SystemResponse],
+    name="List",
+)
+async def ls(  # pylint: disable=invalid-name
+    resource_type: str = "system",
+    db: AsyncSession = Depends(get_async_db),
+) -> List:
+    """Get a list of all of the resources of this type."""
+    return await list_resource(System, db)
+
+
+@system_router.get(
+    "/{fides_key}",
+    dependencies=[
+        Security(
+            verify_oauth_client_prod,
+            scopes=[SYSTEM_READ],
+        )
+    ],
+    response_model=SystemResponse,
+)
+async def get(
+    fides_key: str,
+    resource_type: str = "system",
```

Comment:
Since we're not using this resource_type query param (i see that it's in the base crud get endpoint), should we remove or add an underscore instead for resource_type?

Reply:
yeah, i was just a bit confused as to why we have it there at all in the base crud endpoints, and started getting worried that we rely on it (i.e. maybe some API callers, either the UI or tests, specify the query param?)

Reply:
on some quick testing though things still seem to work fine in the UI, and i'm not seeing any references in the tests, so let me try removing it!

Reply:
My guess was that it was there originally because of how the endpoints are defined in that loop.
They chose to inject the default resource type as a query param, so it wasn't actually intended to be used as a query param by the caller.

Reply:
Oh this is interesting, I am pretty sure the query param is not actually supposed to be used, see this bug:

GET {{host}}/data_category/?resource_type=system

```
[
    {
        "fides_key": "my_test_system",
        "organization_fides_key": "default_organization",
        "tags": null,
        "name": "Test system",
        "description": "Collect data about our users for marketing.",
        "parent_key": null,
        "is_default": false
    }
]

```json

Reply:
good find!